R/as_gt.R

```diff
@@ -397,3 +362,54 @@ footnote_non_binding <- function(x_alpha, full_alpha) {
     "(", a2, " - ", a3, " = ", a1, ") ", "under the null hypothesis."
   )
 }
+
+# where to add the non-binding design footnote
+footnote_row <- function(x, bound) {
+  # for a vector of "Analysis: N", get a logical vector `i`, in which `TRUE`
+  # indicates the position of the largest `N`
+  a <- x$Analysis
+  r <- "^Analysis: ([0-9]+).*"
+  i <- grepl(r, a)
+  k <- as.numeric(sub(r, '\\1', a[i]))
+  i[i] <- k == max(k)
+  i & x$Bound == bound
+}
+
+# a list of information for `as_[gt|rtf].gs_design()` methods
+gs_design_info <- function(
```

Comment:
I am hesitant to use `gs_design_info` as the name, as `info` in gsDesign2 carries a specific meaning – statistical information (1/variance). Can we consider an alternative name for it?

Reply:
The naming of this function is the hardest. I spent nearly half an hour on it but couldn't think of a better name. I ended up using `gsd_parts()`. Please feel free to suggestion a different one.

Reply:
Agree, naming is always the hardest part. As long as we could get rid of the `info` as keywords, we should be good. The new name `gsd_parts` looks good to me.